Dockerfile

```diff
@@ -24,9 +24,10 @@ RUN mkdir -p /packages/shared
 COPY ./packages/shared/package.json /court-backend/packages/shared/package.json
 
 # install dependencies
-RUN npm install
-RUN npx lerna bootstrap
+COPY ./yarn.lock /court-backend/yarn.lock
+RUN yarn install
+RUN yarn lerna link
```

Comment:
running simply `yarn` should be enough I think

Reply:
yeah it's fine, but I prefer install less ambiguous

Reply:
Ah, I think Facu meant that `lerna link` is unnecessary since we're using yarn workspaces now.

Reply:
Yes but I wasn't able to figure out right now why yarn couldn't link and version packages by itself, so opened another issue for this